chemprop/train/evaluate.py

```diff
@@ -26,6 +28,7 @@ def evaluate_predictions(preds: List[List[float]],
     :param dataset_type: Dataset type.
     :param gt_targets: A list of lists of booleans indicating whether the target is an inequality rather than a single value.
     :param lt_targets: A list of lists of booleans indicating whether the target is an inequality rather than a single value.
+    :param quantiles: A list of quantiles for use in pinball evaluation of quantile_interval metric.
```

Comment:
I used the phrase quantile_interval because that is the name of the metric used in the code. It is quantile_interval because it takes in and evaluates an interval using quantile loss.